libsolidity/formal/ModelCheckerSettings.h

```diff
@@ -113,6 +113,7 @@ struct ModelCheckerSettings
 {
 	ModelCheckerContracts contracts = ModelCheckerContracts::Default();
 	ModelCheckerEngine engine = ModelCheckerEngine::None();
+	smtutil::SMTSolverChoice solvers = smtutil::SMTSolverChoice::All();
```

Comment:
But wouldn't `ModelCheckerSettings::operator==()` work implicitly?

Reply:
You mean it would be generated by the compiler for a struct? Unfortunately no. C++20 will have default equality operators but in C++17 it has to be generated manually. Which is a real bummer since it's too easy to forget about updating it.